server/repository/src/migrations/v1_06_00/invoice_add_name_link_id.rs

```diff
@@ -0,0 +1,140 @@
+use crate::{migrations::sql, StorageConnection};
+
+#[cfg(feature = "postgres")]
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        -- Adding invoice.name_link_id
+        ALTER TABLE invoice
+        ADD COLUMN name_link_id TEXT NOT NULL DEFAULT 'temp_for_migration';
+        
+        UPDATE invoice
+        SET name_link_id = name_id;
+        
+        ALTER TABLE invoice ADD CONSTRAINT invoice_name_link_id_fkey FOREIGN KEY (name_link_id) REFERENCES name_link(id);
+        "#,
+    )?;
+    sql!(
+        connection,
+        r#"
+        CREATE OR REPLACE FUNCTION upsert_invoice_changelog()
+        RETURNS trigger AS
+        $$ BEGIN
+            INSERT INTO changelog (table_name, record_id, row_action, name_id, store_id)
```

Comment:
Noting that we're setting `name_link_id` onto `name_id`. Chatted to Andrei, to keep the trigger overhead low it's good to not doing a join on `name_link` to get the `name_id`, better to change `ChangelogRepository` to do the join. Good for another issue, I'll make it.